qubes-rpc/qvm-copy-to-vm.kde

```diff
@@ -19,25 +19,33 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #
 #
+if type kdialog 2> /dev/null; then
+    kdlg=1
+    else kdlg=0
+fi
```

Comment:
Well I had it in mind that there might be an issue assuming that zenity is present, and there might be a need to do a series of tests.